### DISCUSSION TITLE
the size is too big

### DISCUSSION THREAD

**Discussion by Amirulmuuminin** [founded-labs/react-native-reusables#233] [General]
hi, im new to react native and expo. can someone tell me how to reduce the size? i got 65mb for only webview app

**Comment by flomSStaar** [comment]
What dependencies have you installed in your project?

**Reply by Amirulmuuminin** [reply]
no deps, its fresh install create-expo-app

**Reply by Amirulmuuminin** [reply]
Bro, thank you so much for this great work. Now I understand that non play store APK/aab for expo are always big. Then it will become around 10 mb in play store.

Thank you so much for this work ☺️
